### ISSUE TITLE
Can MixOmics integrate different omics data from independent studies_

### ISSUE THREAD

**Issue by ElyasMo** [mixOmicsTeam/mixOmics#231]
Thank you for developing such a great toolkit.

You have mentioned that "Integrative analyses include N-integration with DIABLO (the same N samples are measured on different omics platforms), and P-integration with MINT (the same P omics predictors are measured in several independent studies)." Using mixOmics, would it be possible to integrate the different types of omics data (e.g, transcriptomics, proteomics and metabolomics) which are captured independently in different studies from the same type of human disorder for instance?

In other words, is integrating different types of omics data which are measured independently in different studies possible using mixOmixs?

Regards
Elyas

**Comment by Max-Bladen** [comment]
If I understand you correctly, you are looking to both N-integrate and P-integrate. If so, you're in luck! Have a look at the following functions and pick which one suits you:

-  `mint.block.pls()`
-  `mint.block.spls()`
-  `mint.block.plsda()`
-  `mint.block.splsda()`
